### PR TITLE
Core: Add auth seed for macOS arm64 build 56421

### PR DESCRIPTION
**Pull request by motivewc**
**Changes proposed:**
Quick SQL update for Mac ARM64 Auth Seed

**Issues addressed:**
N/A

**Tests performed:**
Tested in-game 

**Known issues and TODO list:** (add/remove lines as needed)
N/A

### CODE REVIEW

**Comment by motivewc** [comment]
Is there a script to generate a consistent diff-friendly version of auth_database.sql or other base dumps? I'm chaining mine with a sed command from hell to get the newlines right, etc. 👀

File: sql/base/auth_database.sql [hunk 1]
```diff
@@ -3197,7 +3198,8 @@ INSERT INTO `updates` VALUES
 ('2024_08_28_01_auth.sql','BC5D74553AF2D92606F55C1C462D2700FE73BD34','RELEASED','2024-08-28 14:55:05',0),
 ('2024_08_30_00_auth.sql','BD76942F1C29AAA2450E051E7CA552672B5E331B','RELEASED','2024-08-30 19:24:30',0),
 ('2024_08_30_01_auth.sql','0AAC95A9216114177AF0A3A7DE664D061FED21EA','RELEASED','2024-08-30 22:23:08',0),
-('2024_08_31_00_auth.sql','D10F12D25D526A2C5E12FCEA2DEB55C342879F06','RELEASED','2024-08-31 10:32:16',0);
+('2024_08_31_00_auth.sql','D10F12D25D526A2C5E12FCEA2DEB55C342879F06','RELEASED','2024-08-31 10:32:16',0),
+('2024_08_31_01_auth.sql','C415EC73FFBCA85863CB51E276D778CFB352506F','RELEASED','2024-09-01 02:07:49',30);
```
Comment (Ovahlord) [inline sql/base/auth_database.sql:3202]: zero the apply time out pls

**Comment by Aokromes** [comment]
> Is there a script to generate a consistent diff-friendly version of auth_database.sql or other base dumps? I'm chaining mine with a sed command from hell to get the newlines right, etc. 👀

eh i did this initial change of format long long long time ago, i recommend you to search auth_database changes on 3.3.5a branch to find when changed, iirc i added a comment there.

File: sql/updates/auth/master/2024_08_31_01_auth.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+INSERT INTO build_auth_key VALUES (56421, 'Mac', 'A64', 'WoW', 0x5892FFABAEFDCECB0CFFAAA55D2F9B13);
```
Comment (Shauren) [inline sql/updates/auth/master/2024_08_31_01_auth.sql:1]: Add delete like this one

https://github.com/TrinityCore/TrinityCore/commit/cd10553da16bcd99838bd5976939aa314c319b92#diff-a99f5964e5003b34832c13ea6db4f4d02ce121a47dc06c31217e40fd10c9c290R6

File: sql/base/auth_database.sql [hunk 3]
```diff
@@ -3291,10 +3293,6 @@ SET @saved_cs_client     = @@character_set_client;
  1 AS `Permission`*/;
 SET character_set_client = @saved_cs_client;
 
---
```
Comment (Shauren) [inline sql/base/auth_database.sql:3294]: Be more careful when staging changes in git, this shouldn't be in the diff (as well as timestamp at end of file)

**Review by Shauren — CHANGES_REQUESTED** [review]

**Comment by Shauren** [comment]
```
mysqldump -uroot auth --default-character-set='utf8mb4' --routines --result-file sql/base/auth_database.sql
sed -i -e 's$VALUES ($VALUES\n($g' sql/base/auth_database.sql
sed -i -e 's$),($),\n($g' sql/base/auth_database.sql
sed -i -e 's/DEFINER=[^*]*\*/\*/' sql/base/auth_database.sql
sed -i -e 's/utf8mb4_0900_ai_ci/utf8mb4_unicode_ci/g' sql/base/auth_database.sql
```

Is what we use for tdb releases

**Comment by motivewc** [comment]
I came up with this horrific one-liner that does the trick for me in terms of generating consistent dumps of auth_database.sql. Let me know if you see any more problems! 

```bash
docker run -it --rm mysql:8.0.36 mysqldump -h host.docker.internal -u root --hex-blob auth --routines | sed -e 's$VALUES ($VALUES\n($g' |
sed -e 's$),($),\n($g' | sed -e 's/DEFINER=[^*]*\*/\*/' | sed -e 's/utf8mb4_0900_ai_ci/utf8mb4_unicode_ci/g' | sed -e 's/host.docker.internal/localhost/g' | sed -e 's/for Linux (aarch64)/for Linux (x86_64)/g' | sed 's/9\.0\.1/8.0.36-0ubuntu0.22.04.1/g'  | sed -e 's/Dump completed on.*/Dump completed on 2024-05-11  3:06:54/g' > sql/base/auth_database.sql
```

I also have some thoughts as to how to improve the dev experience here for SQL updates. What's the rationale behind requiring auth_database.sql be up to date with each PR's merging, rather than allowing SQL updates to run? 

I understand the desire to keep the database dump "close" to what the current schema looks like, but this does introduce some PR friction. There's also the problem that if another DB change happens between when a PR is opened and merged, it effectively invalidates the other outstanding PRs.

Would you be open to a PR that maybe uses a GitHub Action once weekly to dump a consistent copy of the databases, and opens a PR to merge it in? That could be a nice way to improve the dev experience.

File: sql/base/auth_database.sql (hunk 3 above)
Comment (motivewc) [reply sql/base/auth_database.sql]: You got it.

**Pull request merged**